scripts/run-ios.sh

```diff
@@ -39,41 +39,53 @@ if [ -z "${1-}" ]; then
     exit 1
 fi
 
-SIMULATOR=${1}
+# fetch available iOS Simulators
+xcrun simctl list devices -j > ios_simulators_list.json
 
-# get our desired UUID
-UUID=$(xcrun simctl list devices | grep -E "$SIMULATOR \(" | head -n 1 | awk -F '[()]' '{print $2}')
+SIMULATOR=${1}
 
-# get simulator status
-SIMULATOR_STATE=$(xcrun simctl list devices | grep -E "$SIMULATOR \(" | head -n 1 | awk '{print $NF}')
+# get the first available UDID for Simulators that match the name
+read -r UDID SIMULATOR_STATE IS_AVAILABLE < <(jq --raw-output --arg simulator "$SIMULATOR" '
```

Comment:
This is a nitpick but I prefer if you used curly braces for variables:
```suggestion
read -r UDID SIMULATOR_STATE IS_AVAILABLE < <(jq --raw-output --arg simulator "${SIMULATOR}" '
```
Remember that if you have a string with a veriable expansion it's very easy to create a situation where you get nothing because you did `"$FOObar"` instead of `"${FOO}bar"`. By using curly braces you make sure you avoid this.

Also, it's easier to search for variables in vim using current word search(`*`).

Reply:
Ah yes I missed this. will fix.